app/views/chats/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= turbo_frame_tag "chat_content" do %>
+  <div class="flex flex-col h-full">
+    <div id="chat_messages" class="grow overflow-y-auto">
+      <% @chat.messages.where.not(content: nil).where.not(content: "...").where.not(hidden: true).order(created_at: :asc).each do |message| %>
```

Comment:
If you move the magic string to the `Message` model you could make this a scope to filter it out.